Source/WebCore/rendering/RenderBlockFlow.cpp

```diff
@@ -4461,28 +4466,30 @@ unsigned RenderBlockFlow::computedColumnCount() const
     return 1;
 }
 
-bool RenderBlockFlow::isTopLayoutOverflowAllowed() const
+LayoutBoxExtent RenderBlockFlow::allowedLayoutOverflow() const
 {
-    bool hasTopOverflow = RenderBlock::isTopLayoutOverflowAllowed();
-    if (!multiColumnFlow() || style().columnProgression() == ColumnProgression::Normal)
-        return hasTopOverflow;
-    
-    if (!(isHorizontalWritingMode() ^ !style().hasInlineColumnAxis()))
-        hasTopOverflow = !hasTopOverflow;
+    LayoutBoxExtent allowance = RenderBox::allowedLayoutOverflow();
 
-    return hasTopOverflow;
-}
+    // Handle align-content.
```

Comment:
I don't think the comment adds anything since the next line is about `alignContent()`.